### ISSUE TITLE
UI: A11y: chromecast dialog is not accessible

### ISSUE THREAD

**Issue by ismena** [google/shaka-player#1684]

**Comment by joeyparrish** [comment]
This should be re-tested.  It may have changed in Chrome.  If not, this should be a Chrome bug, since we don't control the Chromecast dialog.

**Comment by joeyparrish** [comment]
Can't reproduce on Chrome 71.
